jsonnet/kube-prometheus/rules/general.libsonnet

```diff
@@ -0,0 +1,19 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'kube-prometheus-general.rules',
+        rules: [
+          {
+            expr: 'count without (instance, pod, node) (up == 1)',
```

Comment:
one last nit, there is usually no space between without and the open parenthesis

Reply:
fixed, ptal